---
title: The repertoire of copy number alteration signatures in human cancer
author:
  - name: Ziyu Tao
    affil: 1-3*
  - name: Shixiang Wang
    affil: 1-3*
  - name: Chenxu Wu
    affil: 1-3*
  - name: Huimin Li
    affil: 1
  - name: Tao Wu
    affil: 1
  - name: Xiangyu Zhao
    affil: 1
  - name: Wei Ning
    affil: 1
  - name: Guangshuai Wang
    affil: 1
  - name: Xue-Song Liu
    affil: 1
    orcid: '0000-0002-7736-0077'
    main: true
    email: liuxs@shanghaitech.edu.cn

    
affiliation:
  - num: 1
    address: School of Life Science and Technology, ShanghaiTech University, Shanghai 201203, China
  - num: 2
    address: Shanghai Institute of Biochemistry and Cell Biology, Chinese Academy of Sciences, Shanghai, China
  - num: 3
    address: University of Chinese Academy of Sciences, Beijing, China
column_numbers: 3
logoleft_name: Figures/shanghaitech_logo.png
logocenter_name: Figures/tigs.png
logoright_name: Figures/liulab.png
output: 
  posterdown::posterdown_html:
    self_contained: false
bibliography: packages.bib
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = FALSE)
```

# Abstract

DNA alteration signatures are recurring patterns that are the imprints of mutagenic processes accumulated during the evolution of cancer cell. Despite the importance of copy number alteration (CNA) in cancer progression,comprehensive understanding about the mutational processes and signatures of CNA is still lacking. Here we developed a method to categorize CNA based on various fragment properties, which reflect the consequences of mutagenic processes and can be extracted from different types of data, including whole genome sequencing (WGS), whole exome sequencing (WES) and SNP array. The signature of CNA has been extracted from 2778 pan-cancer analysis of whole genomes (PCAWG) WGS samples, and further validated using 10851 the cancer genome atlas (TCGA) SNP array dataset. Novel copy number signatures associated with haploid chromosome have been identified. The activities of some copy number signatures consistently predict cancer patientsâ€™ prognosis. This study provides a repertoire for understanding the signatures and mutational processes of CNA.

## Objectives

1. Easy to use reproducible poster design. 
2. Integration with R Markdown.
3. Easy transition from `posterdown` to `pagedown` report or manuscript documents [@R-pagedown].

# Methods

This package uses the same workflow approach as the R Markdown you know and love. Basically it goes from RMarkdown > Knitr > Markdown > Pandoc > HTML/CSS > PDF. You can even use the bibliography the same way [@R-posterdown].

# Results

Usually you want to have a nice table displaying some important results that you have calculated. In `posterdown` this is as easy as using the `kable` table formatting you are probably use to as per typical R Markdown formatting.

You can reference tables like so: Table \@ref(tab:mytable). Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam placerat augue at velit tincidunt semper. Donec elementum porta posuere. Nullam interdum, odio at tincidunt feugiat, turpis nisi blandit eros, eu posuere risus felis non quam. Nam eget lorem odio. Duis et aliquet orci. Phasellus nec viverra est.

```{r mytable, out.width='80%'}
knitr::kable(iris[1:10, 1:4], caption = 'Table caption.',align = 'c',"html")
```

Or with figures: Figure \@ref(fig:standard-plot), or Figure \@ref(fig:morefigs).

```{r standard-plot, out.width='80%', fig.align='center', fig.cap='Great figure!', fig.height=5}
plot(mtcars[1:2])
```

```{r morefigs, out.width='80%', echo=TRUE, fig.cap='Amazing, right?!', fig.height=5}
data <- iris

plot(x = data$Sepal.Length, 
     y = data$Sepal.Width, 
     col = data$Species,
     pch = 19, 
     xlab = "Sepal Length (cm)",
     ylab = "Sepal Width (cm)")

```

# Next Steps

Aliquam sed faucibus risus, quis efficitur erat. Vestibulum semper mauris quis tempus eleifend. Aliquam sagittis dictum ipsum, quis viverra ligula eleifend ut. Curabitur sagittis vitae arcu eget faucibus. In non elementum felis. Duis et aliquam nunc. Nunc pulvinar sapien nunc, vel pretium nisi efficitur in. Fusce fringilla maximus leo et maximus. Fusce at ligula laoreet, iaculis mi at, auctor odio. Praesent sed elementum justo. Aenean consectetur risus rhoncus tincidunt efficitur. Praesent dictum mauris at diam maximus maximus [@R-posterdown].

# Conclusion

Try `posterdown` out! Hopefully you like it!

```{r, include=FALSE}
knitr::write_bib(c('knitr','rmarkdown','posterdown','pagedown'), 'packages.bib')
```

# References
