<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
<meta name="generator" content="pandoc" />
<meta name="viewport" content="width=device-width, initial-scale=1">


<!--
Font-awesome icons ie github or twitter
-->
<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css" integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous">
<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/brands.css" integrity="sha384-n9+6/aSqa9lBidZMRCQHTHKJscPq6NW4pCQBiMmHdUCvPN8ZOg2zJJTkC7WIezWv" crossorigin="anonymous">

<!--
Google fonts api stuff
-->



<title>The repertoire of copy number alteration signatures in human cancer</title>

<script src="PancancerCNAsig_files/header-attrs-2.11/header-attrs.js"></script>





<style>
@page {
size: 45in 38in;
margin: 0;
padding: 0;
}
body {
margin: 0;
font-size: 45px;
width: 45in;
height: 38in;
padding: 0;
text-align: justify;
font-family: Palatino;
}
.poster_wrap {
width: 45in;
height: 38in;
padding: 0cm;
}
.title_container {
width: 45in;
height: calc(38in * 0.15);
overflow: hidden;
background-color: #0b4545;
border: 0 solid #0b4545;
}
.logo_left {
float: left;
width: 10%;
height: 100%;
background-color: #0b4545;
display: flex;
align-items: center;
justify-content: center;
}
.logo_right {
float: right;
width: 10%;
height: 100%;
background-color: #0b4545;
display: flex;
align-items: center;
justify-content: center;
}
.poster_title {
text-align: center;
position: relative;
float: left;
width: 80%;
height: 100%;
color: #FFFFFF;
top: 50%;
transform: translateY(-50%);
-webkit-transform: translateY(-50%);
}
#title {
font-family: Palatino;
}
/* unvisited link */
a:link {
color: #cc0000;
}
.mybreak {
  break-before: column;
}
/* visited link */
a:visited {
color: #cc0000;
}

/* mouse over link */
a:hover {
color: #cc0000;
}

/* selected link */
a:active {
color: #cc0000;
}
.poster_body {
-webkit-column-count: 3; /* Chrome, Safari, Opera */
-moz-column-count: 3; /* Firefox */
column-count: 3;
-webkit-column-fill: auto;
-moz-column-fill: auto;
column-fill: auto;
-webkit-column-rule-width: 1mm;
-moz-column-rule-width: 1mm;
column-rule-width: 1mm;
-webkit-column-rule-style: dashed;
-moz-column-rule-style: dashed;
column-rule-style: dashed;
-webkit-column-rule-color: #0b4545;
-moz-column-rule-color: #0b4545;
column-rule-color: #0b4545;
column-gap: 1em;
padding-left: 0.5em;
padding-right: 0.5em;
height: 100%;
color: #000000
background-color: #ffffff;
}
.poster_title h1 {
font-size: 75pt;
margin: 0;
border: 0;
font-weight: normal;
}
.poster_body_wrap{
width: calc(45in + 0 + 0);
height: calc(38in * 0.83);
padding-top: calc(38in * 0.005);
padding-bottom: calc(38in * 0.01);
background-color: #ffffff;
}
.poster_title h3 {
color: #ffffff;
font-size: 50pt;
margin: 0;
border: 0;
font-weight: normal;
}
.poster_title h3 > sup {
  font-size: 35pt;
  margin-left: 0.02em;
}
.poster_title h5 {
color: #FFFFFF;
font-size: 35pt;
margin: 0;
border: 0;
font-weight: normal;
}
img {
margin-top: 2cm;
margin-bottom: 0;
}
.section {
  padding: 0.2em;
}
.poster_body h1 {
text-align: center;
color: #FFFFFF;
font-size: 65pt;
border: 2mm solid #0b4545;
background-color: #0b4545;
border-radius: 4mm 0mm;
margin-top: 2mm;
margin-bottom: 2mm;
font-weight: normal;
}
.poster_body h2 {
color: #000000;
font-size: 40pt;
padding-left: 4mm;
font-weight: normal;
}
.span {
width: 200%;
}
/* center align leaflet map,
from https://stackoverflow.com/questions/52112119/center-leaflet-in-a-rmarkdown-document */
.html-widget {
margin: auto;
position: sticky;
margin-top: 2cm;
margin-bottom: 2cm;
}
.leaflet.html-widget.html-widget-static-bound.leaflet-container.leaflet-touch.leaflet-fade-anim.leaflet-grab.leaflet-touch-drag.leaflet-touch-zoom {
position: sticky;
width: 100%;
}
pre.sourceCode.r {
background-color: #dddddd40;
border-radius: 4mm;
padding: 4mm;
width: 75%;
/* align-items: center; */
margin: auto;
padding-left: 2cm;
}
code.sourceCode.r{
background-color: transparent;
font-size: 20pt;
border-radius: 2mm;
}
.caption {
font-size: 25pt;
}
.table caption {
font-size: 25pt;
padding-bottom: 3mm;

}
code {
font-size: 1em;
font-family: monospace;
background-color: #00808024;
color: #0b4545;
padding: 1.2mm;
border-radius: 2mm;
}
.poster_title code {
font-size: 1em;
}
table {
font-size: 40px;
margin: auto;
border-top: 3px solid #666;
border-bottom: 3px solid #666;
}
table thead th {
border-bottom: 3px solid #ddd;
}
td {
padding: 8px;
}
th {
padding: 15px;
}
caption {
margin-bottom: 10px;
}
.poster_body p {
margin-right: 4mm;
margin-left: 4mm;
margin-top: 6mm;
margin-bottom: 10mm;
}
.poster_body ol {
margin-right: 4mm;
margin-left: 4mm;
}
#ul {
margin-right: 4mm;
margin-left: 4mm;
}
.references p {
font-size: 20pt;
}
.orcid img {
  width: 1em;
}
</style>
</head>
<body>


<div class="poster_wrap">
<div class="title_container">
<!-- Left Logo  -->
<div class="logo_left">
<img src= 'Figures/shanghaitech_logo.png' style="width: 80%">
</div>
<!-- Poster Title -->
<div class= "poster_title">
<h1 id="title">The repertoire of copy number alteration signatures in human cancer</h1>
<h3 id="author">Ziyu Tao<sup>1-3*</sup>, Shixiang Wang<sup>1-3*</sup>, Chenxu Wu<sup>1-3*</sup>, Huimin Li<sup>1</sup>, Tao Wu<sup>1</sup>, Xiangyu Zhao<sup>1</sup>, Wei Ning<sup>1</sup>, Guangshuai Wang<sup>1</sup>, Xue-Song Liu<sup>1, <a class="orcid" href="https://orcid.org/0000-0002-7736-0077"><img src="https://raw.githubusercontent.com/brentthorne/posterdown/master/images/orcid.jpg"></a></sup></h3><br>
<h5 id="affiliation"><sup>1</sup> School of Life Science and Technology, ShanghaiTech University, Shanghai 201203, China<br> <sup>2</sup> Shanghai Institute of Biochemistry and Cell Biology, Chinese Academy of Sciences, Shanghai, China<br> <sup>3</sup> University of Chinese Academy of Sciences, Beijing, China</h5>
</div>
<!-- Right Logo  -->
<div class="logo_right">
<img src=Figures/liulab.png style="width: 80%">
</div>
</div>

<div class='poster_body_wrap'>
<div class='poster_body'>
<div id="abstract" class="section level1">
<h1>Abstract</h1>
<p>DNA alteration signatures are recurring patterns that are the imprints of mutagenic processes accumulated during the evolution of cancer cell. Despite the importance of copy number alteration (CNA) in cancer progression,comprehensive understanding about the mutational processes and signatures of CNA is still lacking. Here we developed a method to categorize CNA based on various fragment properties, which reflect the consequences of mutagenic processes and can be extracted from different types of data, including whole genome sequencing (WGS), whole exome sequencing (WES) and SNP array. The signature of CNA has been extracted from 2778 pan-cancer analysis of whole genomes (PCAWG) WGS samples, and further validated using 10851 the cancer genome atlas (TCGA) SNP array dataset. Novel copy number signatures associated with haploid chromosome have been identified. The activities of some copy number signatures consistently predict cancer patients’ prognosis. This study provides a repertoire for understanding the signatures and mutational processes of CNA.</p>
</div>
<div id="methodology" class="section level1">
<h1>Methodology</h1>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:figure1"></span>
<img src="Figures/copy%20number%20classification.png" alt="CNA classification strategy for signature analysis." width="70%" />
<p class="caption">
Figure 1: CNA classification strategy for signature analysis.
</p>
</div>
<p>For each CNA segment, the following features have been considered: 1, segment context, including segment shape and copy number change number; 2, Absolute copy number; 3, LOH status; 4, segment size. In total 176 types of CNA segments have been defined accordingly.</p>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:figure2"></span>
<img src="Figures/extraction.png" alt="De novo CNA signature extraction with sigprofiler." width="80%" />
<p class="caption">
Figure 2: De novo CNA signature extraction with sigprofiler.
</p>
</div>
<p>We selected the signature extraction solution as the maximum signature number which meets the following criteria:</p>
<ol style="list-style-type: decimal">
<li><p>No over fit.</p></li>
<li><p>Stability should be at least local maximal.</p></li>
<li><p>Mean cosine distance should be as small as possible.</p></li>
</ol>
<p>Based on the rules, we selected 14 signatures for PCAWG copy number data.</p>
</div>
<div id="results" class="section level1">
<h1>Results</h1>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:figure3"></span>
<img src="Figures/CNS_PCAWG_landscape_proportion_0.05.png" alt="Proportion of tumors with the signature and the median activity of the signature are shown for 32 PCAWG cancer types. For each individual tumor, only signatures that contribute to ≥5% of the total are counted." width="80%" />
<p class="caption">
Figure 3: Proportion of tumors with the signature and the median activity of the signature are shown for 32 PCAWG cancer types. For each individual tumor, only signatures that contribute to ≥5% of the total are counted.
</p>
</div>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:figure4"></span>
<img src="Figures/pcawg_cox.png" alt="CNA signature activity and cancer patients’ prognosis." width="70%" />
<p class="caption">
Figure 4: CNA signature activity and cancer patients’ prognosis.
</p>
</div>
<p>The CNA signatures extracted from cancer patients could be cancer prognosis biomarkers, to test this possibility, multivariate Cox (corrected for cancer types and each CNA signature) regression analyses were performed to evaluate the associations between each genome alteration signature and cancer patients’ survival time.In pan-cancer level, activities of CNS11, CNS12 are significantly associated with poor OS.</p>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:figure5"></span>
<img src="Figures/CNS-Profile.png" alt="Representative CNA profile, prominent features and potential mechanisms for each identified CNA signatures extracted in PCAWG dataset." width="90%" />
<p class="caption">
Figure 5: Representative CNA profile, prominent features and potential mechanisms for each identified CNA signatures extracted in PCAWG dataset.
</p>
</div>
</div>
<div id="conclusion" class="section level1">
<h1>Conclusion</h1>
<p>Here we developed a unified and comprehensive method for copy number signature analysis. Our method can be applied in cancer patients with copy number profiles generated with WGS, WES or SNP array data. Our copy number signature analysis method is based on a novel and comprehensive method to catalog copy number segments. New CNA patterns have been identified, and the activities of some CNA signature have been demonstrated to have prognostic power in pan-cancer datasets, suggesting copy number signatures could be biomarkers to guide cancer precision medicine.</p>
</div>
<div id="acknowledgments" class="section level1">
<h1>Acknowledgments</h1>
<p>We thank ShanghaiTech University High Performance Computing Public Service Platform for computing services. This work was supported by The National Natural Science Foundation of China (31771373), Shanghai Science and Technology Commission (21ZR1442400), and startup funding from ShanghaiTech University.</p>
</div>
<div id="recent-works-of-out-lab" class="section level1">
<h1>Recent works of out lab</h1>
<ul>
<li><p>He, Zaoke, et al. “Ggct (γ‐glutamyl cyclotransferase) plays an important role in erythrocyte antioxidant defense and red blood cell survival.” British Journal of Haematology 195.2 (2021): 267-275.</p></li>
<li><p>Wang, Shixiang, et al. “Copy number signature analysis tool and its application in prostate cancer reveals distinct mutational processes and clinical outcomes.” PLoS genetics 17.5 (2021): e1009557.</p></li>
<li><p>Wang, Shixiang, et al. “UCSCXenaShiny: an R/CRAN package for interactive analysis of UCSC xena data.” Bioinformatics (2021).</p></li>
<li><p>Wang, Shixiang, et al. “Sigflow: an automated and comprehensive pipeline for cancer genome mutational signature analysis.” Bioinformatics 37.11 (2021): 1590-1592.</p></li>
</ul>
<p><a href="https://github.com/XSLiuLab/">©Cancer Biology Group</a> 2021</p>
<p><small>Research group led by Xue-Song Liu in ShanghaiTech. University. Lab website is shown in QR code at bottom right.</small></p>
</div>
</div>
</div>

</div>



</body>
</html>
